source/about/orchestration.rst

```diff
@@ -39,55 +37,52 @@ Branding                                      Yes         Yes
-      - [Configure SSL for Mattermost](https://about.mattermost.com/ssl-configuration/) 
+      - [Configure SSL for Mattermost](https://about.mattermost.com/ssl-configuration/)
       - [Configure SMTP email for Mattermost](https://about.mattermost.com/smtp-configuration/)
 
-
 2. **Unofficial deployment options should be documented**. Unofficial deployment configurations, such as use of Linux operating systems that are not officially supported, should be documented in the README.
 
-
 Security Updates 
-~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~
 
 1. **Document commitment to providing security updates.** README or equivalent states `We highly recommend users subscribe to the Mattermost security updates email list. When notified of a security update, the maintainers of this deployment solution will make an effort to update to the secure version within 10 days.`
 
-2. **Commit to making an effort for your deployment to provide the latest security update within 10 days of announcement**. This is typically a one-line change. 
+2. **Commit to making an effort for your deployment to provide the latest security update within 10 days of announcement**. This is typically a one-line change.
 
 Branding 
-~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~
 
-1. **Support Mattermost branding and naming guidelines**. To ensure naming is clear across deployment solutions, README or equivalent should contain ``The name for this deployment solution in the context of [Mattermost branding guidelines](https://mattermost.org/brand-guidelines/) is `[NAME] for Mattermost by [CREATOR]`.`` For example, ``Multi-node Docker deployment solution for Mattermost by John Doe``. This is the name that will be used to refer to your work in Mattermost community materials. 
+1. **Support Mattermost branding and naming guidelines**. To ensure naming is clear across deployment solutions, README or equivalent should contain ``The name for this deployment solution in the context of [Mattermost branding guidelines](https://mattermost.org/brand-guidelines/) is `[NAME] for Mattermost by [CREATOR]`.`` For example, ``Multi-node Docker deployment solution for Mattermost by John Doe``. This is the name that will be used to refer to your work in Mattermost community materials.
 
 Upgrade 
-~~~~~~~~~~~~~~~~~~~~~~~
-
-1. **Support upgrade of Mattermost**. Enable user interface or commandline upgrade of a Mattermost deployment to latest version based on `upgrade procedure when Mattermost is embedded <https://docs.mattermost.com/developer/integration-faq.html#how-should-i-automate-the-install-and-upgrade-of-mattermost-when-included-in-another-application>`__
+~~~~~~~
 
+1. **Support upgrade of Mattermost**. Enable user interface or command line upgrade of a Mattermost deployment to latest version based on `upgrade procedure when Mattermost is embedded <https://docs.mattermost.com/developer/integration-faq.html#how-should-i-automate-the-install-and-upgrade-of-mattermost-when-included-in-another-application>`__
```

Comment:
This link resolves to a docs.mm.com page that isn't included in the LHS. Has this content been migrated to the devs.mm.com site?